property.go

```diff
@@ -0,0 +1,91 @@
+package opslevel
+
+import "fmt"
+
+type PropertyDefinitionInput struct {
+	Name   string `json:"name"`
+	Schema string `json:"schema"`
+}
+
+type PropertyDefinition struct {
+	Aliases []string `graphql:"aliases"`
+	Id      ID       `graphql:"id"`
+	Name    string   `graphql:"name"`
+	Schema  string   `graphql:"schema"`
+}
+
+type PropertyDefinitionConnection struct {
+	Nodes      []PropertyDefinition
+	PageInfo   PageInfo
+	TotalCount int
```

Comment:
```suggestion
	TotalCount int `graphql:"-"`
```
We cannot rely on TotalCount from the API because they don't always calculate it the same due to filter and such.  So we should make this field not be requested and then build up our down total count value based on `len(Nodes)`